### PR TITLE
Update A1.csv

### PR DESCRIPTION
**Pull request by ahaminsgithub**
add heat feedback (Qb in kW) to burner

### CODE REVIEW

File: Hamins_Gas_Burners/A1.csv [hunk 1]
```diff
@@ -1,8 +1,8 @@
...
-0.208,0,0.203,0.095652
-0.344,0,0.393,0.0052174
-0.6,0,0.668,0.0078261
-NaN,NaN,1.088,0.0030435
+m,kW/m2,m,kW/m2,kW
+r,Q(r),z,Q(z),Qb
+0.081,0.0039,0.002,0.077826,0.17
+0.13,0,0.088,0.29217,NaN
+0.208,0,0.203,0.095652,NaN
+0.344,0,0.393,0.0052174,Nan
```
Comment (rmcdermo) [inline Hamins_Gas_Burners/A1.csv:6]: @ahaminsgithub @mcgratta Will "Nan" cause a problem?

**Comment by mcgratta** [comment]
I did not notice that, but probably not.

>

**Comment by mcgratta** [comment]
We can easily do a global search and replace, plus convert from dos to unix.

>

File: Hamins_Gas_Burners/A1.csv (hunk 1 above)
Comment (drjfloyd) [reply Hamins_Gas_Burners/A1.csv:6]: We have NaN in some of the verification exact csv files like water_evaporation_4.csv.

Comment (rmcdermo) [reply Hamins_Gas_Burners/A1.csv:6]: Right, I was worried about the lower case "n" for that one particular instance "Nan".

**Comment by ahaminsgithub** [comment]
oops, I'll make sure that doesn't happen again

**Pull request merged**